### ISSUE TITLE
Gradient Boosting baseline

### ISSUE THREAD

**Issue by kamil271e** [iwamaciek/stwp#12]
Baseline model using gradient boosting algorithm such as LightGBM/XGBoost #4 should be updated (handle grid point neighbors) before addressing this issue.
